### PR TITLE
Update demo gists

### PR DESCRIPTION
**Pull request by NSDCars5**
Made new demo gists which work with updated origami-lib.

### CODE REVIEW

**Comment by AvaisP** [comment]
Hey
Before I review this, are you a GCI student?

**Comment by NSDCars5** [comment]
@AvaisP Yes, I am.

**Comment by AvaisP** [comment]
Please ping me on Gitter. 
I'll need to create a GCI task for this so that you can claim it :D

**Comment by NSDCars5** [comment]
@AvaisP I messaged you on Gitter.

File: docs/library.rst [hunk 1]
```diff
@@ -120,13 +120,13 @@ Output functions
 
 			Array/Tuple of "local path of images on the disk" in text
 
-			An example can be seen at this `gist <https://gist.github.com/tocttou/591d28bb89641ba7b94783687be65fdb>`_.
+			An example can be seen at this `gist <hhttps://gist.github.com/NSDCars5/e7b9af826c8055e6949ea9092e3be343>`_.
```
Comment (AvaisP) [inline docs/library.rst]: Wrong url

Comment (NSDCars5) [reply docs/library.rst]: Fixed. @AvaisP

**Comment by NSDCars5** [comment]
@AvaisP Changes have been made.

**Pull request merged**